TF_Tooling_Web/src/Connection/TouchFreeServiceTypes.ts

```diff
@@ -91,6 +102,16 @@ export class HandPresenceEvent {
     }
 }
 
+// Class: InteractionZoneEvent
+// This data structure is used to receive interaction zone requests
+export class InteractionZoneEvent {
```

Comment:
🤷 Consider interface instead. Alternatively the code can at least be shortened:
```ts
export class InteractionZoneEvent {
    constructor(public state: InteractionZoneState) {}
}
```

Reply:
Changed to an interface :+1: